### PR TITLE
Add Reload to glossary

### PR DESCRIPTION
**Pull request by dannytsang**
## Proposed change
There are references to reload in various places such as:
* Glossary page
* Scenes integration
* [Automation services](https://www.home-assistant.io/docs/automation/services/)

so I thought adding the term to the glossary makes sense and not an obvious meaning behind the word.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_data/glossary.yml [hunk 1]
```diff
@@ -282,6 +282,18 @@
     other integrations.
   link: /docs/configuration/platform_options/
 
+- term: Reload
+  definition: >-
+    Updates Home Assistant configuration files. Changes are normally
+    automatically updated. However, when changes are made outside of the UI
+    and at a file level, Home Assistant is not aware of these changes and
+    require the configuration file(s) to be reloaded to pick up any changes
```
Comment (c0ffeeca7) [inline source/_data/glossary.yml]:
```suggestion
    requires the configuration file(s) to be reloaded to pick up any changes
```

File: source/_data/glossary.yml [hunk 2]
```diff
@@ -282,6 +282,18 @@
...
+    made by going to *YAML configuration reloading* section in
+    **Developer tools > YAML**.
```
Comment (c0ffeeca7) [inline source/_data/glossary.yml]:
```suggestion
    **Developer tools** > **YAML**.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Thank you for the explanation, @dannytsang. I find this very helpful.
Question: is this YAML reload the same as using **Settings** > **System** > **Quick reload**?

**Comment by dannytsang** [comment]
> Thank you for the explanation, @dannytsang. I find this very helpful. Question: is this YAML reload the same as using **Settings** > **System** > **Quick reload**?

That's a good point. I see that as equivalent of reload option *All YAML Configuration*. Let me add the quick reload in because I think that is a more user friendly and the developer tools is more advanced.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @dannytsang :+1:

File: source/_data/glossary.yml [hunk 3]
```diff
@@ -282,6 +282,19 @@
     other integrations.
   link: /docs/configuration/platform_options/
 
+- term: Reload
+  definition: >-
+    Updates Home Assistant configuration files. Changes are normally
```
Comment (frenck) [inline source/_data/glossary.yml:287]: The first sentence is actually incorrect and misleading.

It states: "Updates Home Assistant configuration files", but it doesn't update or touch those at all...

Comment (dannytsang) [reply source/_data/glossary.yml:287]: When you put it like that, you're right it doesn't make changes to the configuration files. I'll raise a new PR.

**Pull request merged**